std/stdio.d

```diff
@@ -5579,7 +5579,7 @@ private size_t readlnImpl(FILE* fps, ref char[] buf, dchar terminator, File.Orie
                 app.putonly(p[0 .. i]);
             }
             fp._cnt -= i;
-            fp._ptr += i;
+            () @trusted { fp._ptr += i; }();
```

Comment:
These 2 lines are a bit of a mystery to me, `fp` is not read again. Should I just remove them?

Reply:
Nevermind, I forgot about `goto`!